contrib/kube-prometheus/docs/How-to-monitor-an-external-secure-etcd-cluster.md

```diff
@@ -0,0 +1,164 @@
+# How to monitor a secure external etcd service with Prometheus Operator
+This guide will help you monitor an external etcd cluster. When the etcd is not hosted inside Kubernetes.
+This is often the case with the Kubernetes setup. This has been tested with kube-aws but same principals will apply to other tools.
+
+# Step 1 - Make the etcd certificates available to Prometheus pod
+Prometheus Operator (and Prometheus) allow us to specify a tlsConfig. This is required as most likely your etcd metrics end points is secure.
+
+## a - Create the secrets in the namespace 
+Prometheus Operator allows us to mount secrets in the pod. By loading the secrets as files, they can be made available inside the Prometheus pod.
+
+`kubectl -n monitoring create secret generic etcd-certs --from-file=CREDENTIAL_PATH/etcd-client.pem --from-file=CREDENTIAL_PATH/etcd-client-key.pem --from-file=CREDENTIAL_PATH/ca.pem`
+
+where CREDENTIAL_PATH is the path to your etcd client credentials on your work machine. 
+(Kube-aws stores them inside the credential folder).
+
+## b - Get Promnetheus Operator to load the secret
+In the previous step we have named the secret 'etcd-certs'.
+
+Edit prometheus-operator/contrib/kube-prometheus/manifests/prometheus/prometheus-k8s.yaml and add the secret under the spec of the Prometheus object manifest:
```

Comment:
As you mention the entire manifest below, I think you can skip mentioning this file

Reply:
I didn't get that. I only put half of the manifest